tests/smoke/smoke_test.go

```diff
@@ -9,170 +10,233 @@ import (
 	"time"
 
 	opb "github.com/accuknox/auto-policy-discovery/src/protobuf/v1/observability"
+	wpb "github.com/accuknox/auto-policy-discovery/src/protobuf/v1/worker"
 	"github.com/accuknox/auto-policy-discovery/src/types"
 	"github.com/kubearmor/discovery-engine/tests/util"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/rs/zerolog/log"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
 	nv1 "k8s.io/api/networking/v1"
-	"sigs.k8s.io/yaml"
 )
 
 var stopChan chan struct{}
 
-func checkPath(mp []types.KnoxMatchPaths, str string) string {
-	for i := range mp {
-		if mp[i].Path == str {
-			return str
-		}
+var grpcClient *grpc.ClientConn
+
+var (
+	expectedFilePaths = []string{
+		"/dev/urandom",
+		"/etc/ld.so.cache",
 	}
-	return ""
-}
+	expectedFileDirs = []string{
+		"/lib/x86_64-linux-gnu/",
+		"/var/www/html/",
+		"/usr/src/wordpress/",
+	}
+	expectedProcessPaths = []string{
+		"/usr/sbin/apache2",
+		"/usr/local/bin/php",
+	}
+	expectedProcessDir = []string{
+		"/bin/",
+		"/usr/bin/",
```

Comment:
have we confirmed that we are getting these process execution without exec-ing into the pod?